core/src/test/java/io/questdb/std/fastdouble/AbstractFloatNumericallyGeneratedTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * @(#)NumericallyGeneratedTest.java
+ * Copyright © 2021. Werner Randelshofer, Switzerland. MIT License.
+ */
+
+package io.questdb.std.fastdouble;
+
+import io.questdb.std.NumericException;
+import org.junit.Test;
+
+import java.util.Random;
+
+import static org.junit.Assert.assertEquals;
+
+abstract class AbstractFloatNumericallyGeneratedTest {
+    /**
+     * Seed for random number generator.
+     * Specify a literal number to obtain repeatable tests.
+     * Specify System.nanoTime to explore the input space.
+     * (Make sure to take a note of the seed value if
+     * tests failed.)
+     */
+    public static final long SEED = System.nanoTime();
```

Comment:
`AbstractDoubleNumericallyGeneratedTest` has a constant seed value:
```java
public static final long SEED = 0;//System.nanoTime();
```
while this and `AbstractLexicallyGeneratedTest` tests use nanoTime. Would be nice to use either a constant or nanoTime in all of these tests. If we decide to use nanoTime, we should print the seed value into stdout to be able to reproduced potential test failures.

Reply:
Addressed in 2ff64e43467de473b9a567492ee0d63ba6e9e81d